.github/CODEOWNERS

```diff
@@ -149,7 +149,7 @@
 /cmd/agent/dist/conf.d/containerd.d/    @DataDog/container-integrations
 /cmd/agent/dist/conf.d/container_image.d/      @DataDog/container-integrations
 /cmd/agent/dist/conf.d/container_lifecycle.d/  @DataDog/container-integrations
-/cmd/agent/dist/conf.d/jetson.d/        @DataDog/agent-platform
+/cmd/agent/dist/conf.d/jetson.d/        @DataDog/@DataDog/agent-ci-experience
```

Comment:
Note: to be updated later, it doesn't make sense for this to be owned by any of the ex-Agent Platform teams.

Reply:
Posted in `datadog-agent` slack to find someone to adopt it